src/main/java/com/znsio/teswiz/runner/Driver.java

```diff
@@ -482,4 +483,18 @@ public void injectMediaToBrowserstackDevice(String uploadFileURL) {
                     "injectMediaToBrowserstackDevice is not implemented for: " + cloudName);
         }
     }
+
+    public static void scrollInDynamicLayer(Driver driver, String direction) {
+        Dimension dimension = driver.getInnerDriver().manage().window().getSize();
```

Comment:
why do you need to pass Driver to a method in the Driver?

Reply:
removed now